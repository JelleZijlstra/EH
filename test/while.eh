#!/usr/bin/ehi
# Basic while loop functionality
ctr = 0
while ctr < 5
	set ctr++
	$echo ctr
endwhile
