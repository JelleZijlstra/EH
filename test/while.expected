%%stdout%%
1
2
3
4
5
%%stderr%%
